monitor/check_dir_empty

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2017 SUSE LLC  All rights reserved.
+#
+# check_dir_empty is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation, version 2 of
```

Comment:
Should this be version 3?

Reply:
Nope, needs to be v2 for nagios compatibility